jetty-client/src/test/java/org/eclipse/jetty/client/ConnectionPoolHelper.java

```diff
@@ -30,6 +29,6 @@ public static Connection acquire(AbstractConnectionPool connectionPool, boolean
 
     public static void tryCreate(AbstractConnectionPool connectionPool, int pending)
```

Comment:
This helper is only used for testing, so its signature should have been changed to reflect the new `ConnectionPool.tryCreate(boolean)` one.